docs/content/users/quickstart.md

```diff
@@ -196,6 +196,25 @@ The legacy type `drupal` will be interpreted as the latest stable version of Dru
     ddev launch $(ddev drush uli)
     ```
 
+=== "Drupal CMS"
+
+    ```bash
+    mkdir my-drupal-cms && cd my-drupal-cms
+    ddev config --project-type=drupal11 --docroot=web
+    ddev composer create --stability="RC" drupal/cms
+    ddev launch
+    ```
+
+    or use the Zip file download technique:
+
+    ```
+    CMS_VERSION=1.0.0-rc2
+    curl -o drupal-cms.zip -fL https://ftp.drupal.org/files/projects/cms-${CMS_VERSION}.zip
+    unzip drupal-cms.zip
```

Comment:
I guess I think we should let people handle these complexities rather than adding lots of things to it that they might not understand and that might be error-prone. I definitely don't think we should be doing an `rm -rf drupal-cms`

Reply:
but in the current form you have a zip file lingering in the directory root you started the quickstart in, compared to other quickstarts we dont have a new working directory the project is located in, and the curl command creates a drupal-cms sub directory, plus you have the example of the backdrop quickstart that is applying those steps. those are many complexities i agree but at the same time i would try to stay consistent in betweent quickstarts and that is why i've suggested adjusting the drupal cms quickstart to the backdrop one.

Reply:
The curl command does not create a subdirectory. But the unzip does. 

```suggestion
    unzip drupal-cms.zip && rm drupal-cms.zip
```